tests/nested/core20/tpm/task.yaml

```
@@ -24,7 +12,8 @@ execute: |
     execute_remote "test -e /sys/kernel/security/tpm0/binary_bios_measurements"
 
     echo "and secure boot is enabled on the nested vm"
-    retry -n 5 --wait 5 execute_remote "test-snapd-mokutil --sb-state" | MATCH "SecureBoot enabled"
+    execute_remote "ls /sys/firmware/efi/efivars/SecureBoot-*"
```

hmm this is not correct, because the SecureBoot variable can be there but not enabled. We should be checking something like:

```suggestion
    execute_remote "xxd /sys/firmware/efi/efivars/SecureBoot-8be4df61-93ca-11d2-aa0d-00e098032b8c" | MATCH "00000000: 0600 0000 01\s+....."
```

on a system with secure boot enabled we have:
```
$ xxd /sys/firmware/efi/efivars/SecureBoot-8be4df61-93ca-11d2-aa0d-00e098032b8c
00000000: 0600 0000 01                             .....
```

and on a system without secure boot enabled we have:

```
$ xxd /sys/firmware/efi/efivars/SecureBoot-8be4df61-93ca-11d2-aa0d-00e098032b8c 
00000000: 0600 0000 00                             .....
```

Also I confirmed that xxd is available in the core20 snap so this is fine to do.

---

@anonymouse64 thanks for the check, I'll update the test accordingly